src/MICore/LaunchOptions.cs

```diff
@@ -444,6 +444,20 @@ public string VisualizerFile
             }
         }
 
+        private bool _waitDynamicLibLoad;
+        /// <summary>
+        /// If true, wait for dynamic library load to finish.
+        /// </summary>
+        public bool WaitDynamicLibLoad
+        {
+            get { return _waitDynamicLibLoad; }
+            set
+            {
+                VerifyCanModifyProperty("StopOnDynamicLibLoad");
```

Comment:
What happens if this is false? Do we still go through this codepath in the default scenarios? Does this throw and block debugging?